config/examples/stm32h5-tz-dualbank.config

```diff
@@ -0,0 +1,31 @@
+ARCH?=ARM
```

Comment:
Can you please add the new configs to the build test `.github/workflows/test-configs.yml`?